<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddSlaveChromatogramm</class>
 <widget class="QWidget" name="DialogAddSlaveChromatogramm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,1">
     <item>
      <widget class="QLabel" name="captionLabel">
       <property name="text">
        <string>Add slave chromatogramm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Choose files to add for current markup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="CustomCheckBox" name="checkBox">
       <property name="text">
        <string>Choose all</string>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="treeView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ui/controls/customcheckbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
